Makefile

```diff
@@ -34,3 +34,20 @@ generate-mix32 :
 doctest :
 	perl -i -e 'while (<ARGV>) { print unless /package-id base-compat-\d+(\.\d+)*/; }' .ghc.environment.*
 	doctest src
+
+native:
+	nix-build -A native.splitmix
+
+freebsd:
+	nix-build -A freebsd.splitmix
+
+android:
+	nix-build -A android.splitmix
+
+js:
+	nix-build -A js.splitmix
+
+wasm:
```

Comment:
Not in nixpkgs [yet](https://github.com/NixOS/nixpkgs/pull/225000), so we're using ghc-wasm-meta for now.